src/ripple/app/misc/HashRouter.h

```diff
@@ -84,12 +84,18 @@ class HashRouter
             flags_ |= flagsToSet;
         }
 
-        void swap (std::set <PeerShortID>& other)
+        std::set <PeerShortID>&& peers()
```

Comment:
It's a private class, so only callers that know what's going on should touch it, but I don't object to a rename.